### ISSUE TITLE
About replace in ReLU and Dropout

### ISSUE THREAD

**Issue by TomMao23** [zhanghang1989/PyTorch-Encoding#399]
I noticed that a lot of your code is written like this
```python
nn.ReLU(True),
nn.Dropout(0.1, False),
```
why you use inplace operator in activation fuction but not in dropout? Does this have any special meaning?
